app/src/main/java/fr/free/nrw/commons/bookmarks/pictures/BookmarkPicturesFragment.java

```diff
@@ -37,11 +36,12 @@ public class BookmarkPicturesFragment extends DaggerFragment {
     private GridViewAdapter gridAdapter;
     private CompositeDisposable compositeDisposable = new CompositeDisposable();
 
-    @BindView(R.id.statusMessage) TextView statusTextView;
-    @BindView(R.id.loadingImagesProgressBar) ProgressBar progressBar;
-    @BindView(R.id.bookmarkedPicturesList) GridView gridView;
-    @BindView(R.id.parentLayout) RelativeLayout parentLayout;
+    TextView statusTextView;
+    ProgressBar progressBar;
+    GridView gridView;
+    RelativeLayout parentLayout;
```

Comment:
Remove this TextView, ProgressBar, GridView, RelativeLayout. We have direct access to it as we are using the view binding.